addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1485,6 +1485,35 @@ QUnit.module('basic_fields', {
         list.destroy();
     });
 
+    QUnit.test('text fields edit mode inside list have correct height just after init', function (assert) {
+        assert.expect(2);
+
+        this.data.partner.records[0].txt = "a\nb\nc\nd\ne\nf";
+
+        var list = createView({
+            View: ListView,
+            model: 'partner',
+            data: this.data,
+            arch: '<list editable="top">' +
+                    '<field name="foo"/>' +
+                    '<field name="txt"/>' +
+                '</list>',
+        });
+
+        // click to enter edit
+        list.$('.o_data_cell:first').click();
+        var $textarea = list.$('textarea:first');
+
+        // make sure the correct data is there
+        assert.strictEqual($textarea.val(), this.data.partner.records[0].txt);
+
+        // make sure there is no scroll bar
+        assert.strictEqual($textarea.innerHeight(), $textarea[0].scrollHeight,
+            "textarea should not have a scroll bar");
+
+        list.destroy();
+    });
```

Comment:
this test seems very similar to the one of the previous commit

Reply:
The first test relies on focus (existing working event to trigger the resize) to just check the size is correct.

The second test specifically tests by clicking on another column, without using the focus event. The goal is not directly to test the size, but that the resize is indeed called in that case.

Reply:
I'll add a bit of comment to both tests to make this more clear.